Otherwise fw_printenv build fails with "arm-linux-gcc command unknown" 
with external toolchains.

Should be removed when migrating to latest BR, as now fw_printenv build
is done in packages/

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/boot/u-boot/u-boot.mk
===================================================================
--- buildroot.orig/boot/u-boot/u-boot.mk	2011-10-04 22:09:23.000000000 +0200
+++ buildroot/boot/u-boot/u-boot.mk	2011-10-04 22:10:19.000000000 +0200
@@ -212,7 +212,7 @@
 # Build manually fw_printenv for the target
 $(TARGET_DIR)/usr/sbin/fw_printenv: $(U_BOOT_DIR)/.configured
 	$(TARGET_CONFIGURE_OPTS) \
-		$(MAKE) HOSTCC="$(TARGET_CC)" -C $(U_BOOT_DIR) env
+		$(MAKE) HOSTCC="$(TARGET_CC)" CROSS_COMPILE="$(TARGET_CROSS)" -C $(U_BOOT_DIR) env
 	$(INSTALL) -m 0755 -D $(U_BOOT_DIR)/tools/env/fw_printenv $@
 	ln -sf fw_printenv $(TARGET_DIR)/usr/sbin/fw_setenv
 
